### ISSUE TITLE
Document how to get complete document distribution from LDA

### ISSUE THREAD

**Issue by Antupis** [RaRe-Technologies/gensim#2489]
Document how to to get full document distribution from LDA, that would help in those cases where you are implementing your own distance algorithm and you need full  distribution. Personally I set  minimum_probability=0.0 but there might be other ways.

**Comment by piskvorky** [comment]
Yes, this is a common request. We should document the `minimum_probability=0.0` + `sparse2full()` for numpy array more prominently.

@Antupis can you open a PR with your suggested solution (updated docs)?

**Comment by Hiyorimi** [comment]
Can I finish this one?

**Comment by mpenkov** [comment]
@Hiyorimi Sure, please go ahead.

**Comment by Hiyorimi** [comment]
@mpenkov take a look please. I didn't get whole BoW part? What do should I describe in this example?

**Comment by Hiyorimi** [comment]
@piskvorky Can you take a look please?
